### PR TITLE
feat: Modify feature request form

### PR DESCRIPTION
**Pull request by wallyrb**
Signed-off-by: Wally Rodriguez B <wallyrb@fb.com>

## Summary

This PR attempts to move away from asking partners what feature they need.  Instead, we focus in understanding the problem and asking what problem they are trying to solve and decide how we want to best solve it. 

Note, this PR is re-naming the form from "Feature Request" to "Feedback Form"

## Test Plan

Below snapshot of the before and after of this form.

**Before:**
image

**After:**

image

### CODE REVIEW

**Comment by wallyrb** [comment]
@ekowtaylor could you help me reviewing this and approving if no further comments?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -14,25 +14,21 @@ assignees: amarpad, ekowtaylor, talkhasib
 - **Affected Subcomponent:** Orchestrator pod/service or AGW service
 - **Deployment Environment:** e.g. AWS (orc8r), local docker-compose (orc8r), bare-metal (AGW), Vagrant (AGW)
 
-### Describe the Feature
+### Describe the problem
 
-**Is your feature request related to a known issue? Please describe.**
-A clear and concise description of what the problem is and provide links to any relevant issues
+**Description**
+A clear and concise description of what is the problem, under which scenarios or how frequent it happen
```
Comment (ekowtaylor) [inline .github/ISSUE_TEMPLATE/feature_request.md]: @wallyrb small nit, should it read 'what is the problem' or 'what the problem is'?

Comment (wallyrb) [reply .github/ISSUE_TEMPLATE/feature_request.md]: > @wallyrb small nit, should it read 'what is the problem' or 'what the problem is'?

@ekowtaylor thanks, you are right is an indirect question. I changed it. Let me know if you have any other comments

**Review by themarwhal — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -14,25 +14,21 @@ assignees: amarpad, ekowtaylor, talkhasib
...
+A clear and concise description of what the problem is, under which scenarios or how frequent it happen
 
-**Specify the applicable product use-cases**
-Magma product use-case support is growing, specify what use-cases apply including FWA, CWF, PLTE, MBB etc.
+**Additional context**
+Add any other context or screenshots about the problem here. You can also provide links to relevant resources.
 
-**Describe the solution you'd like**
-A clear and concise description of your preferred solution including links to design docs/proposals will be great.
+### Describe your idea(optional)
```
Comment (hcgatewood) [inline .github/ISSUE_TEMPLATE/feature_request.md:25]: @wallyrb missing space after "idea" and before the paren 🙂

Comment (wallyrb) [reply .github/ISSUE_TEMPLATE/feature_request.md:25]: I missed that, thanks @hcgatewood! Here the PR https://github.com/magma/magma/pull/8968 for fixing that

**Pull request merged**